tlslite/messages.py

```diff
@@ -19,6 +19,7 @@
 from .x509certchain import X509CertChain
 from .utils.tackwrapper import *
 from .extensions import *
+from .utils.format_input import noneAsUnknown
```

Comment:
So `format_output` is appropriate name? Ive got problem with creating suitable names for files/variable...